update_box86.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+cd /home/pi/box86
+sudo cp /usr/local/bin/box86 box86/backup
```

Comment:
Why `sudo`? You can copy anything you have read access to -- which you do have here.